From ef6dc2583233c3b1a4a59d36033f724166bd07eb Mon Sep 17 00:00:00 2001
From: Lauri Kopo <lauri.kopo@jolla.com>
Date: Tue, 28 Jan 2020 11:53:50 +0200
Subject: [PATCH] fix_systemd_path

---
 restorecond/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/restorecond/Makefile b/restorecond/Makefile
index 12452cd2..27062bc2 100644
--- a/restorecond/Makefile
+++ b/restorecond/Makefile
@@ -48,8 +48,8 @@ install: all
 	install -m 644 restorecond.desktop $(DESTDIR)$(AUTOSTARTDIR)/restorecond.desktop
 	-mkdir -p $(DESTDIR)$(DBUSSERVICEDIR)
 	install -m 644 org.selinux.Restorecond.service  $(DESTDIR)$(DBUSSERVICEDIR)/org.selinux.Restorecond.service
-	-mkdir -p $(DESTDIR)$(SYSTEMDDIR)/system
-	install -m 644 restorecond.service $(DESTDIR)$(SYSTEMDDIR)/system/
+	-mkdir -p $(DESTDIR)$(SYSTEMDDIR)/
+	install -m 644 restorecond.service $(DESTDIR)$(SYSTEMDDIR)/
 relabel: install
 	/sbin/restorecon $(DESTDIR)$(SBINDIR)/restorecond 
 
-- 
2.24.1

